apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -74,114 +71,6 @@ const useBrandColors = ({ brandColor, darkBrandColor }: { brandColor?: string; d
 };
 
 type BookingPageProps = BookPageProps | TeamBookingPageProps | HashLinkPageProps;
-const BookingFields = ({
```

Comment:
Reusing BookingFields component that's created for new booker

Reply:
Should we make these fixes into the v2 booker instead? Since this will be deprecated.

Reply:
It's there as well. BookingFields is shared.